src/timer.jquery.js

```diff
@@ -69,7 +70,14 @@
 	 * Render pretty time
 	 */
 	function render() {
-		$el[display](secondsToTime(totalSeconds));
+		if (options.countdown && duration > 0)
+		{
```

Comment:
Can you please follow the file's pattern to use the first curly brace of a 'if' statement on the same line as 'if'. This is to ensure consistency.